community/front-end/ofe/website/ghpcfe/models.py

```diff
@@ -1230,11 +1230,10 @@ class GCPFilestoreFilesystem(Filesystem):
 
     FILESTORE_TIER = (
         ("u", "TIER_UNSPECIFIED"),
```

Comment:
What is the behavior when the tier is unspecified?

Reply:
@heyealex 

Terraform creation fails, as expected with the same validator in #746, and filesystem is not created.
<pre>
Error: Invalid value for variable

  on main.tf line 27, in module "fs1":
  27:   filestore_tier       = "TIER_UNSPECIFIED"
    ├────────────────
    │ var.filestore_tier is "TIER_UNSPECIFIED"
</pre>

Would agree this is a bit pointless in the Front End - was probably added for completeness.

Reply:
Could it be removed as well without breaking anything? I can see the need for a null value, but a user shouldn't be able to select it.

Reply:
I just removed it and ran a quick test, everything worked as expected.